delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -167,8 +167,11 @@ def _prepare_customer(self, picking):
         if picking.picking_type_id.code != "outgoing":
             partner = picking.partner_id
 
+        partner_name = partner.name
+        if not partner_name:
+            raise exceptions.UserError(_("Customer name is required."))
```

Comment:
Should the raised error be more explicit ?
My thinking is, I mark a transfer as done, the label is printed in the process, this error comes on my screen...

Reply:
bah, this is what's done for every other field required by postlogistics.
What would you do?

Reply:
:smile: